### PR TITLE
Add documentation of list of ports used

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Add documentation of list of ports used
# Purpose
To make it easier to manage the ports used
# Contents
Add documentation of list of ports used
Bring bts envoy from [QuickMPC-BTS](https://github.com/acompany-develop/QuickMPC-BTS/blob/main/Config/envoy.yaml)
# Testing Methods Performed

### CODE REVIEW

File: docs/list-of-port.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+|  [computation_envoy_container](https://github.com/acompany-develop/QuickMPC/blob/main/config/computation_container/compute1/envoy.yaml)   |  [computation_container](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/computation_container#readme)    |  50020, 51020  |  http           |  gRPC              |  内部通信  |
+|  [computation_container](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/computation_container#readme)    |  [bts_envoy](https://github.com/acompany-develop/QuickMPC/blob/main/config/beaver_triple_service/envoy.yaml)  |  50020         |  https or http  |  gRPC              |           |
+|  [bts_envoy](https://github.com/acompany-develop/QuickMPC/blob/main/config/beaver_triple_service/envoy.yaml)  |  [beaver_triple_service](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/beaver_triple_service#readme)        |  54000         |  http           |  gRPC              |  内部通信  |
+
+補足
+
+* プロトコルがhttps or htttpの通信に関しては、ssl証明書の管理方法によりプロトコルが異なる。
+    * ssl証明書を用いない場合→http
+    * LBでssl終端を持つ場合→http
+    * envoyコンテナでssl終端を持つ場合→https
```
Comment (mdonaka) [inline docs/list-of-port.md:21]: @KotaTakahashi9320 
Depending on your preference, how about the following changes?
```suggestion
## 補足
プロトコルがhttps or htttpの通信に関しては、ssl証明書の管理方法によりプロトコルが異なる。
* ssl証明書を用いない場合→http
* LBでssl終端を持つ場合→http
* envoyコンテナでssl終端を持つ場合→https
```

Comment (KotaTakahashi9320) [reply docs/list-of-port.md]: @mdonaka 
Thank you!
It looked better there!

**Review by mdonaka — APPROVED** [review]
Thanks for the addition!

**Comment by mdonaka** [comment]
@KotaTakahashi9320 
Could you rebase this branch?

**Comment by KotaTakahashi9320** [comment]
@mdonaka 
I rebased it!
Please check.

File: docs/list-of-port.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+|  [manage_container](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/manage_container#readme)     |  [manage_envoy_container](https://github.com/acompany-develop/QuickMPC/blob/main/config/manage_container/manage1/envoy.yaml)   |  50010         |  https or http  |  gRPC              |           |
+|  [manage_envoy_container](https://github.com/acompany-develop/QuickMPC/blob/main/config/manage_container/manage1/envoy.yaml)   |  [manage_container](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/manage_container#readme)     |  50011, 51011  |  http           |  gRPC              |  内部通信  |
+|  [manage_container](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/manage_container#readme)     |  [computation_container](https://github.com/acompany-develop/QuickMPC/tree/main/packages/server/computation_container#readme)    |  50010         |  http           |  gRPC              |  内部通信  |
```
Comment (come2ry) [inline docs/list-of-port.md:10]: @KotaTakahashi9320 
こちらmc -> cc_envoyではなくmc -> ccで合っていますか？(念の為確認)

Comment (KotaTakahashi9320) [reply docs/list-of-port.md:10]: @come2ry 
Envoy is now used when communicating with the outside world.
Should it also be used for internal communications?

**Pull request merged**